src/main/java/ui/UiRenderer.java

```diff
@@ -0,0 +1,71 @@
+package ui;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+// uncomment below line when merging with Taskmanager
+// import main.TaskManager;
+
+public class UiRenderer {
+    private static final String[] WEEK_DAYS = {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"};
+
+    public static void printWeekHeader() {
+        for (String day : WEEK_DAYS) {
+            System.out.print("+------------");
```

Comment:
Borders for the calendar could be replaced with String literals for better code clarity! LGTM

Reply:
Well said, I will take note to include a macro in the next code iteration!